runelite-client/src/main/java/net/runelite/client/game/ItemMapping.java

```diff
@@ -296,6 +312,7 @@ public enum ItemMapping
 	ITEM_GHARZI_RAPIER(GHRAZI_RAPIER, HOLY_GHRAZI_RAPIER),
 	ITEM_MASTER_SCROLL_BOOK(MASTER_SCROLL_BOOK_EMPTY, MASTER_SCROLL_BOOK),
 	ITEM_ARCANE_SIGIL(ARCANE_SIGIL, ELIDINIS_WARD_F, ELIDINIS_WARD_OR),
+	ITEM_GUARDIAN_BOOTS(GUARDIAN_BOOTS, ECHO_BOOTS),
```

Comment:
I wouldn't call this a "revertible item", as the echo crystal is lost when it is reverted. I think it would fit better under the "Degradable/charged weaponry/armour" section